services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -722,6 +738,87 @@ public void processDlqRequestForwarded() {
 
     }
 
+    @Test
+    void processCCDReferenceRequestToSubmitSuccess() {
```

Comment:
Probably also need to add a test where the request is not one of the valid types.

Reply:
done